src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -375,3 +375,26 @@ The following example illustrates the contexts part of the <Link to="/sdk/event-
   }
 }
 ```
+
+## Current Culture Context
```

Comment:
maybe it'd make sense to move some stuff out from the device context to this one then, like `timezone`

Reply:
as timezone now belongs to culture as well, lets deprecate the one in the device context?

Reply:
See 
- https://github.com/getsentry/develop/pull/329#issuecomment-836457964
- https://github.com/getsentry/develop/pull/329#issuecomment-836485242

Reply:
deprecating is more like, adding a note that this is also available under the new culture type, not really removing.